scribbler/tests/test_views.py

```diff
@@ -11,6 +11,16 @@
 from . import DaysLog
 from .base import ScribblerDataTestCase, Scribble
 
+import time
```

Comment:
Minor nit but can we try to keep these imports somewhat ordered: standard lib, Django, internal code in alpha order